src/machine/board_thumby.go

```diff
@@ -0,0 +1,94 @@
+//go:build thumby
+
+// This contains the pin mappings for the Thumby.
+//
+// https://thumby.us/
+package machine
+
+import (
+	"device/rp"
+	"runtime/interrupt"
+)
+
+const (
+	THUMBY_SCK_PIN = I2C1_SDA_PIN
+	THUMBY_SDA_PIN = I2C1_SCL_PIN
+
+	THUMBY_CS_PIN    = GPIO16
+	THUMBY_DC_PIN    = GPIO17
+	THUMBY_RESET_PIN = GPIO20
+
+	THUMBY_LINK_TX_PIN = UART0_TX_PIN
+	THUMBY_LINK_RX_PIN = UART0_RX_PIN
+	THUMBY_LINK_PU_PIN = GPIO2
+
+	THUMBY_BTN_LDPAD_PIN = GPIO3
+	THUMBY_BTN_RDPAD_PIN = GPIO5
+	THUMBY_BTN_UDPAD_PIN = GPIO4
+	THUMBY_BTN_DDPAD_PIN = GPIO6
+	THUMBY_BTN_B_PIN     = GPIO24
+	THUMBY_BTN_A_PIN     = GPIO27
+
+	THUMBY_AUDIO_PIN = GPIO28
+
+	THUMBY_SCREEN_RESET_PIN = GPIO20
+)
+
+// I2C pins
+const (
+	I2C0_SDA_PIN Pin = NoPin
+	I2C0_SCL_PIN Pin = NoPin
+
+	I2C1_SDA_PIN Pin = GPIO18
+	I2C1_SCL_PIN Pin = GPIO19
+)
+
+// SPI pins
+const (
+	SPI0_SCK_PIN = GPIO18
+	SPI0_SDO_PIN = GPIO19
+	SPI0_SDI_PIN = GPIO16
+
+	SPI1_SCK_PIN = NoPin
+	SPI1_SDO_PIN = NoPin
+	SPI1_SDI_PIN = NoPin
+)
+
+// Onboard crystal oscillator frequency, in MHz.
+const (
+	xoscFreq = 12 // MHz
+)
+
+// USB CDC identifiers
+const (
+	usb_STRING_PRODUCT      = "Thumby"
```

Comment:
so, when I was at Google, _ in names was strongly discourated. As, in the sense, rejected. I notice these all over tinygo but maybe it's time to become more idiomatic. Otherwise it starts to look like "C in Go" -- that was the argument made to me.

also, convention has it that usb should USB, as it is an acronym. 

USBProductName and USBManufacturerName might be ok, but why not just ProductName and or Product and Manufacturer? 

I understand there are conventions you have, so maybe you don't want to change at this point, but these identifier styles would be deemed non-idiomatic ...

If you don't want to export, then you could say _USBProduct or something? 

Same for VID and PID.

Reply:
The names that we have chosen in the past have been generally trying to match datasheets just to have better searchability for non-Go people.

If we want to make a chance to be more in line with Go norms, which we probably want to do, we should most likely have a bigger conversation on how to make that happen.

Reply:
@deadprogram Agreed- I'd strongly suggest against making identifier name changes/decisions until we have an automated way of "correcting" these names- as in a tool that can be run from the command line to avoid code churn and much back and forth on opinions.